src/dyn_model.jl

```diff
@@ -68,16 +69,20 @@ function DynamicPowerManagementProblem(
 
     # Define a storage variable for n nodes over T timesteps
     s = [Variable(n) for _ in 1:T]
+    # Define a charge and discharge variable for n nodes over T timesteps
+    ch = [Variable(n) for _ in 1:T]
+    dis = [Variable(n) for _ in 1:T]
 
+    #TODO: need to handle the initial condition
     subproblems = vcat(
```

Comment:
It looks like this can be reduced to a single line:
```
subproblems = [PowerManagementProblem(fq[t], fl[t], d[t], pmax[t], gmax[t], A, B; ch=ch[t], dis=dis[t]) for t in 1:T]
```